quiz-question/question5.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quiz Buzz</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="stylesheet" href="../css/style.css">
+    <link rel="stylesheet" href="../css/navbar.css">
+    <link rel="stylesheet" href="./question.css">
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <span class="brand-name">QUIZ BUZZ</span>
+            <ul class="nav-links">
+                <li>
+                    <a href="../index.html" class="nav-item">Home</a>
+                    <p class="active"></p>
+                </li>
+                <li>
+                    <a href="../quiz-category/quiz-category.html" class="nav-item">Category</a>
+                    <p class="active"></p>
+                </li>
+                <li>
+                    <a href="./rules.html" class="nav-item">Rules</a>
+                    <p class="active"></p>
+                </li>
+                <li>
+                    <a href="#" class="nav-item">About us</a>
+                    <p class="active"></p>
+                </li>
+            </section>
+        </nav>
+    </header>
+    <main>
+        <div class="question-container">
```

Comment:
If possible you can add comments for readability then it will be perfect

Reply:
sure will do that